debug/index.html

```diff
@@ -326,7 +330,6 @@ <h1 class="title">Transmux Analyzer</h1>
 
       reader.addEventListener('loadend', function() {
         var bytes = new Uint8Array(reader.result);
-
```

Comment:
You keep removing whitespace! The careful use of whitespace (in this case, after variable definitions) adds some readability to the code.